tools/rimage/src/rimage.c

```diff
@@ -204,6 +204,12 @@ int main(int argc, char *argv[])
 		goto out;
 	}
 
+	if (image.num_modules > MAX_MODULES) {
```

Comment:
If not limited by an enum or bitfield, then lest future proof with a larger number.

Reply:
This should really be the size of the IPC4 container for module ID, although lets set around 256 if IPC4 uses an uint32_t since this is an array in rimage today IIRC.

Reply:
@lgirdwood  This value was selected by developer. It results from the average number of modules descriptors with configuration structures that will fit in the manifest. The maximum size of a manifest is 8 KB in CAVS 1.5. Now this size is bigger on a new platforms. I think it would be best if we convert it into dynamic size array.

Reply:
ok, we can do dynamic array as an incremental upgrade.